src/preprocessing.jl

```diff
@@ -217,14 +251,26 @@ end
 
 tag_pos!(entity) = error("Not yet implemented")
 
+"""
+    sparse_terms(crps [, alpha])
 
+Find the sparse terms from Corpus, occuring in less than `alpha` percentage of the documents, with `alpha = 0.05` by default.
```

Comment:
Instead writing `alpha = 0.05 by deafult` in the description, maybe I can mention that in function signature (?)

Reply:
Good idea! This is what the official documentation suggests:

> Optional arguments should be represented with their default values (i.e. `f(x, y=1)`) when possible, following the actual Julia syntax. Optional arguments which do not have a default value should be put in brackets (i.e. `f(x[, y]`) and `f(x[, y[, z]])`)

Reply:
:+1: